index.html

```diff
@@ -1,304 +1,416 @@
-                <h2>Profesional Art <br />Printing Data</h2>
-                <p>
-                    A daily selection of privately personalized reads; no accounts or
-                    sign-ups required. Has been the industry's standard.
-                </p>
-                <ul>
-                    <li>
-                        <a href="#">HTML</a>
-                    </li>
-                    <li>
-                        <a href="#">BOOTSTRAP</a>
-                    </li>
-                    <li>
-                        <a href="#">RUBY</a>
-                    </li>
-                </ul>
-                <button type="button">See project</button>
-            </article>
+          <section class="content-desc">
+            <img src="images/icons/pro_sec.svg" alt="Content">
+            <div class="top-left1">
+              <h2>Data Dashboard<br/>Healthcare</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul>
+              <li>
+                <a href="#">HTML</a>
+              </li>
+              <li>
+                <a href="#">BOOTSTRAP</a>
+              </li>
+              <li>
+                <a href="#">RUBY</a>
+              </li>
+            </ul>
+            <button type="button">See project</button>
+          </div>
+          </section>
         </section>
 
         <section class="center-content">
-            <article class="content-desc">
-                <h2>Profesional Art <br />Printing Data</h2>
-                <p>
-                    A daily selection of privately personalized reads; no accounts or
-                    sign-ups required. Has been the industry's standard.
-                </p>
-                <ul>
-                    <li>
-                        <a href="#">HTML</a>
-                    </li>
-                    <li>
-                        <a href="#">BOOTSTRAP</a>
-                    </li>
-                    <li>
-                        <a href="#">RUBY</a>
-                    </li>
-                </ul>
-                <button type="button">See project</button>
-            </article>
+          <section class="content-desc">
+            <img src="images/icons/Data_content_image.svg" alt="">
+            <div class="top-left">
+              <h2>Data Dashboard<br/>Healthcare</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul>
+              <li>
+                <a href="#">HTML</a>
+              </li>
+              <li>
+                <a href="#">BOOTSTRAP</a>
+              </li>
+              <li>
+                <a href="#">RUBY</a>
+              </li>
+            </ul>
+            <button type="button">See project</button>
+          </div>
```

Comment:
- [ ] Great work on your project cards! One small suggestion: **it was mentioned by previous reviewers that adding a hover effect to your project cards would make your portfolio even better,** and closer to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=DOmVpbk2buQv96CX-0). 🎯
 
  _Here is a video example to help you understand what effect you need to add and make your portfolio stand out even more!_ 🎬

[scrnli_5_6_2023_5-30-23 PM.webm](https://user-images.githubusercontent.com/107952724/236648961-166eca8f-769a-44b9-94db-cc65b9853c63.webm)